workflows/compare_clinical/config_ba1.yaml

```diff
@@ -0,0 +1,38 @@
+api_url: "https://lapis.cov-spectrum.org/open/v2/sample/aggregated"
+country: "Switzerland"
+wastewater_data_path: "../../../LolliPop/lollipop_covvfit/deconvolved.csv"
```

Comment:
I think for now we should merge this, but at some point we should standardize the paths across our environments (see #30).